frontend/src/app/Main.tsx

```diff
@@ -44,7 +44,37 @@ const Main: React.FC<ViewProps> = () => {
       }}
       renderDrawerContent={() => {
         return (
-          <Text style={SpacingStyle.pad(drawerInsets, 16)}>Drawer content</Text>
+          <View style={SpacingStyle.pad(drawerInsets, 16)}>
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="accessible" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>ADA Ride Request</Text>
+                </View>
+            </TouchableOpacity>
+            
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="error-outline" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>Upcoming Outages</Text>
```

Comment:
Upcoming Alerts?

Reply:
Alert kind of implies to me that it's *now*, so saying upcoming doesn't make a whole lot of sense. But I agree outages might be weird too.

Maybe "Upcoming Delays"?

Reply:
But alerts encompass delays and outages, so it's a little inaccurate to use one word or the other. Not sure what to do here.